Gum/Managers/NameVerifier.cs

```diff
@@ -79,6 +79,25 @@ public bool IsElementNameValid(string componentNameWithoutFolder, string folderN
             }
             return string.IsNullOrEmpty(whyNotValid);
         }
+        internal bool IsCategoryNameValid(string name, ElementSave categoryContainer, out string whyNotValid)
+        {
+            string standardizedName = Standardize(name);
+
+            string existingName = null;
+            StateSaveCategory existing = categoryContainer.GetStateSaveCategoryRecursively(item =>
+            {
+                if (Standardize(item.Name) == standardizedName)
+                {
+                    existingName = item.Name;
+                    return true;
+                }
+
+                return false;
+            });
+
+            whyNotValid = $"Cannot add category — a category with the name {existingName} is already defined in {categoryContainer}";
```

Comment:
This should probably only assign whyNotValid if there is an existing item. Otherwise this may cause confusion when debugging.